.github/workflows/e2e.jdk-versions.yaml

```diff
@@ -47,6 +47,8 @@ jobs:
           git submodule sync --recursive
           git -c protocol.version=2 submodule update --init --force --recursive --depth=1
       - name: Build Docker Image
+        env:
+          JAVA_VERSION: ${{ matrix.jdk }}
```

Comment:
So, agent and backend tests will run in the same JDK versions, right?

Reply:
> So, agent and backend tests will run in the same JDK versions, right?

Yes, the agent and backend should be independent in JDK versions, so just to make it simple